{"sid": "cd8be63c17fd4be28f5ab0f987863874", "uid": null, "qid": "54b57fe8cf0a4ef891f6af836426747b", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [54b57fe8cf0a4ef891f6af836426747b]\nTraceback (most recent call last):\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/action_dispatcher.py\", line 139, in process\n    output = self.workerContext.execution.execute(data, self.ray, self.state, update)\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/application_interface.py\", line 66, in execute\n    self.execution_function(model)\n  File \"/openfabric/app/main.py\", line 92, in execute\n    agent = Agent('gemini',\"TEST_USER\",session_id ,generator)\n  File \"/openfabric/app/Agent/agent.py\", line 27, in __init__\n    Agent._sessionsManager = SessionManager()\nTypeError: __init__() takes 0 positional arguments but 1 was given\n", "created_at": "2025-06-30T09:45:29.466891"}], "status": "FAILED", "created_at": "2025-06-30T09:45:12.788471", "updated_at": "2025-06-30T09:45:29.466916", "finished": true}